src/Package/TestApp/Utility/ViewTreeDump.cs

```diff
@@ -36,27 +35,36 @@ internal class ViewTreeDump {
         }
 
         private static void CompareVisualTree(VisualTreeObject actual, VisualTreeObject expected, bool compareProperty = true) {
+            bool visible = true;
+            
             // compare
-            actual.Name.ShouldBeEquivalentTo(expected.Name);
+            actual.Name.Should().BeEquivalentTo(expected.Name);
```

Comment:
Please don't use `BeEquivalentTo()` for equality comparisons. It's meant for object graphs in very specific circumstances. `Be()` is more appropriate here.